website/content/en/docs/building-operators/helm/migration.md

```diff
@@ -39,7 +39,7 @@ Let's create the same project but with the Helm plugin:
 ```sh
 $ mkdir nginx-operator
 $ cd nginx-operator
-$ operator-sdk init --plugins=helm.operator-sdk.io/v1 --domain=com --group=example --version=v1alpha1 --kind=Nginx
+$ operator-sdk init --plugins=helm.sdk.operatorframework.io/v1 --domain=com --group=example --version=v1alpha1 --kind=Nginx
```

Comment:
Ideally we would direct users to use short names to get the latest non-default plugin:
```suggestion
$ operator-sdk init --plugins=helm --domain=com --group=example --version=v1alpha1 --kind=Nginx
```

Reply:
I'll fix this in a follow-up.